AUTHORS

```diff
@@ -181,7 +181,6 @@ Ryan Petrello <lists@ryanpetrello.com>
 Fuji Goro <gfuji@cpan.org>
 Siddharth Mahendraker <siddharth_mahen@hotmail.com>
 Dave Pacheco <dap@joyent.com>
-Mathias Buus <m@ge.tt>
```

Comment:
@aashil that's the way I'd prefer it. When in doubt, ask the individual what they would prefer :-)